technologies/job/spark/spark-aws-3.1/innerContexts/jre/spark-aws-3.1-jre-11/build.me

```diff
@@ -0,0 +1 @@
+08/07/2022
```

Comment:
What's the meaning of this date ? 
if it's the creation of the image by us, why is it not the current date (20/09/2022)?

Reply:
It's a hack when you want to force the build, not necessary on this PR I can delete the file